File: test-scripts/linter-tests/bad.swift
import Foundation

// Long line violation
let veryLongVariableName = "This is a very long string that exceeds the recommended line length limit and should be wrapped"

// Force unwrapping
let name: String? = "John"
print(name!)

// Unused variable
let unusedVariable = 42

// Trailing whitespace   
let value = 10  

// Missing documentation
class MyClass {
    var property: String
    
    init(property: String) {
        self.property = property
    }
    
    // Force cast
    func doSomething(obj: Any) {
        let str = obj as! String
        print(str)
    }
}

// Multiple empty lines


// Violation
func badFunction( ){
    print("Bad spacing")
}

// Implicit getter
class AnotherClass {
    var name: String {
        get {
            return "name"
        }
    }
}

File: test-scripts/linter-tests/bad.py
#!/usr/bin/env python3
"""Bad Python file for Pylint testing"""

import os
import sys
import json  # Unused import

# Missing module docstring for some functions

def test_function():
    x = 1
    y = 2  # Unused variable
    return x

class TestClass:
    def __init__(self):
        self.value = 1
    
    def method(self, arg1):
        """Method with unused argument"""
        return self.value

# Line too long
very_long_line = "This is a very long line that exceeds the maximum line length and should trigger a pylint warning about line length conventions"

# Too many blank lines




def another_function():
    pass

# Invalid name (should be snake_case)
InvalidName = "test"

# Missing final newline
